partner_view_adj/views/res_partner_views.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_partner_tree_view" model="ir.ui.view">
+        <field name="name">res.partner.tree.inherit</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_tree" />
+        <field name="arch" type="xml">
+            <field name="user_id" position="before">
+                <field name="property_product_pricelist" optional="show" />
```

Comment:
Please take this chance to add `team_id` and `property_payment_term_id` as well, with `optional="hide"`.

Please also update the search view to include these fields (please always do this whenever we add fields to the list).